src/main/resources/locales/en/alphabetical-search.json

```diff
@@ -5,7 +5,7 @@
   "tableHeader1": "Court or tribunal",
   "nationalListsHeader": "National lists",
   "sjpLink": "Single Justice Procedure cases",
-  "filter": "Filter",
+  "filter": "Filter by Jurisdiction / Region",
```

Comment:
did this need applying to court name search too or did you see it only needs to be in 1 of the 2 places?

Reply:
I've done this now, it wasn't in the ticket but it does make sense to do both.